src/styles/styles.js

```diff
@@ -350,6 +350,10 @@ const styles = {
         paddingLeft: 0,
     },
 
+    pl6: {
+        paddingLeft: 6,
+    },
+
```

Comment:
Don't really need these if we follow my suggestion above, also since the paddingLeft styles aren't being used anywhere else.

Reply:
I'm also curious how this one might work with `utilities/spacing.js` in that the number after the `pl` typically denotes a multiplier which multiplies our default spacing unit of four:

<img width="320" alt="image" src="https://user-images.githubusercontent.com/2319350/211745012-2c4d21e0-bed4-48db-93f2-d5f34471454c.png">

Reply:
yeah, using those naming conventions `pl6` definitely isn't accurate here, it doesn't mean `paddingLeft: 6px` rather `paddingLeft: 24px`, which is why I also wanted to get rid of that class